commons-numbers-complex-arrays/src/test/java/org/apache/commons/numbers/complex/arrays/ComplexListTest.java

```diff
@@ -223,6 +224,71 @@ void testReplaceAllComplexScalarFunction() {
         Assertions.assertEquals(objectList, actualList);
     }
 
+    @Test
+    void testReplaceAllComplexConsumer() {
```

Comment:
`testForEach...`

This test can be parameterized for size. This will eliminate duplication:
```Java
@ParameterizedTest
@ValuesSource(ints = {0, 10})
void testForEachComplexConsumer(int size) {
    List<Complex> data1 = generateList(size);
```